src/data/permissions/task.ts

```diff
@@ -49,24 +24,26 @@ export default function loadModule(
     WORK_REQUEST_CREATED: { inherits: 'send-task-work-request' },
     WORKER_ASSIGNED: { inherits: 'set-task-assignee' },
     WORKER_UNASSIGNED: { inherits: 'remove-task-assignee' },
-    'cancel-task': { inherits: 'is-founder-or-admin' },
-    'create-task': { inherits: 'is-founder-or-admin' },
-    'finalize-task': { inherits: 'is-founder-or-admin' },
-    'remove-task-assignee': { inherits: 'is-founder-or-admin' },
-    'send-task-work-invite': { inherits: 'is-founder-or-admin' },
+    'cancel-task': { inherits: 'is-admin' },
+    'create-task': { inherits: 'is-admin' },
+    'finalize-task': { inherits: 'is-admin' },
+    'remove-task-assignee': { inherits: 'is-admin' },
+    'send-task-work-invite': { inherits: 'is-admin' },
     'send-task-work-request': async () => true,
-    'set-task-assignee': { inherits: 'is-founder-or-admin' },
-    'set-task-domain': async (userAddress, { event }) => {
-      // Ideally, this would also check the current domainId; this has to
-      // be done outside of the action controller, because Orbit only passes
-      // the current entry to the `canAppend` method.
-      return event && event.type === EventTypes.DOMAIN_SET
-        ? isAdminOrFounder(userAddress, event.payload.domainId)
-        : false;
-    },
-    'set-task-due-date': { inherits: 'is-founder-or-admin' },
-    'set-task-payout': { inherits: 'is-founder-or-admin' },
-    'set-task-skill': { inherits: 'is-founder-or-admin' },
-    'update-task': { inherits: 'is-founder-or-admin' },
+    'set-task-assignee': { inherits: 'is-admin' },
+    // disallow changing of the domain for now
+    'set-task-domain': async () => false,
```

Comment:
I have a feeling this is going to frustrate a lot of folks. Is it worth exploring a solution with product team that makes it more obvious during task creation that the task will be "permanently" (at least for now) assigned to the currently selected domain? Perhaps that would require less effort than a long-term DLP solution?

Reply:
As as stop gap solution for this... how about?

We have a _New Task_ button, which, upon clicking, will open a dropdown/tooltip, with all the available domains for you to select from.

That will make it a tiny bit more obvious to the user the domain that task is going to be created in.

It's not the best solution, but it's better then what we have now.

Thoughts? @chmanie @ceolson01 and @JoinColony/product

Reply:
I agree that this is not ideal, but we also want to be done with this at some point. I suggest to leave it be for now and figure out a better solution together with @JoinColony/product, then implement it later.

Reply:
Actually, that could be interesting. I could imagine we just add an ellipses to the button's copy so it's more clear that there are other options:

`New Task …` and that opens the menu with

`in root`
`in development`
`in product`

I'd be okay with that as an initial solution.

---

I'm also okay with leaving it as is for now and:

a. Letting users discover this, delete it, then recreate it in the correct domain
b. Letting users complain to us that it sucks so we can fix it then